sourcecode/shared/src/main/scala/sourcecode/SourceContext.scala

```diff
@@ -105,6 +106,15 @@ object Pkg extends SourceCompanion[String, Pkg](new Pkg(_)){
   def impl(c: Compat.Context): c.Expr[Pkg] = Impls.enclosing[Pkg](c)(_.isPackage)
 }
 
+case class Caller[A](value: A)
```

Comment:
I am not keen on this being parametric, it would be best if it was a `String` to avoid leaking runtime information.